core-aam/core-aam.html

```diff
@@ -1208,7 +1208,12 @@
 				<tr id="role-map-scrollbar">
 					<th><a class="role-reference" href="#scrollbar"><code>scrollbar</code></a></th>
           <td><code>ROLE_SYSTEM_SCROLLBAR</code></td>
-          <td><code>scrollbar</code></td>
+          <td>
+            <ul>
+              <li>Control Type = <code>ScrollBar</code></li>
```

Comment:
Nit:  The core-aam currently uses the style:
"Control type is `ScrollBar`"

That is, no "=" sign.  I prefer consistency with the current style.  Any objections?